editoast/src/views/path/path_item_cache.rs

```diff
@@ -276,7 +276,7 @@ fn collect_path_item_ids(path_items: &[&PathItemLocation]) -> (Vec<String>, Vec<
 }
 
 /// Retrieve operational points from operational point uic codes
-async fn retrieve_op_from_uic(
+pub async fn retrieve_op_from_uic(
```

Comment:
Instead of making them `pub`, I'm wondering if we should instead move these functions in a place where it makes more sense. If they're used in a bunch of different places inside `views`, then maybe their place is in `views/mod.rs`? Maybe @flomonster ([who is the last to have moved these functions around](https://github.com/OpenRailAssociation/osrd/pull/8035/files#diff-a9e8dffdf0b65df8459d64f474aafbb40d8647101c57673e6e874dcb8ea8f861R184-R241)) or @younesschrifi ([who introduce the function](https://github.com/OpenRailAssociation/osrd/pull/6753/files#diff-5af3da79c94b1ba9c091e3569d129e730831804df9bc4f24dbcab8736addada2R243-R286)) have an opinion on that?